patches/tModLoader/Terraria/ModLoader/ModKeybind.cs

```diff
@@ -11,13 +12,16 @@ namespace Terraria.ModLoader
 	public class ModKeybind
```

Comment:
We should make this extend `ModType` and be autoloadable, but that is perhaps outside the scope of this PR.